### ISSUE TITLE
Position lights/sidelights

### ISSUE THREAD

**Issue by StreetFonso** [user-grinch/ModelExtras#20]
Add a position lights/sidelights function (common in large vehicles) with a dummy that accepts settings in the same way as other lights (_prmFFFF00056, for example). This light should turn on at night, just like the headlights and taillights. The material is not necessary, as there is already the light_night_material R0G16B255. It is important that the direction of the shadow takes into account the rotation of the dummy.

**Comment by user-grinch** [comment]
Doesn't night lights do the same thing?

**Comment by StreetFonso** [comment]
No, because there is no setting for the shadow to be only to the sides. The "LIGHT_TYPE_DIRECTIONAL" (0, 1 or 2) is forward, backward, forward and sideways or backward and sideways. Position lights/sidelights must also have the shadow only to the right and left. The image below shows the position lights of Linerunner that needs shadows only to left.
![gallery4](https://github.com/user-attachments/assets/1fd830f3-3d9e-4fcf-be25-419f31f4edff)
